src/objects/Contact_Language__c.object

```diff
@@ -0,0 +1,115 @@
+    <actionOverrides>
+        <actionName>Edit</actionName>
+        <type>Default</type>
+    </actionOverrides>
+    <actionOverrides>
+        <actionName>Follow</actionName>
+        <type>Default</type>
+    </actionOverrides>
+    <actionOverrides>
+        <actionName>List</actionName>
+        <type>Default</type>
+    </actionOverrides>
+    <actionOverrides>
+        <actionName>New</actionName>
+        <type>Default</type>
+    </actionOverrides>
+    <actionOverrides>
+        <actionName>SaveEdit</actionName>
+        <type>Default</type>
+    </actionOverrides>
+    <actionOverrides>
+        <actionName>Tab</actionName>
+        <type>Default</type>
+    </actionOverrides>
+    <actionOverrides>
+        <actionName>View</actionName>
+        <type>Default</type>
+    </actionOverrides>
+    <allowInChatterGroups>false</allowInChatterGroups>
+    <compactLayoutAssignment>HEDA_Contact_Language_Compact_Layout</compactLayoutAssignment>
+    <compactLayouts>
+        <fullName>HEDA_Contact_Language_Compact_Layout</fullName>
+        <fields>Name</fields>
+        <label>HEDA Contact Language Compact Layout</label>
+    </compactLayouts>
+    <deploymentStatus>Deployed</deploymentStatus>
+    <description>Connection between a Language and a Contact for storing details about a Contact&apos;s language fluency.</description>
+    <enableActivities>false</enableActivities>
+    <enableBulkApi>true</enableBulkApi>
+    <enableFeeds>false</enableFeeds>
+    <enableHistory>false</enableHistory>
+    <enableReports>true</enableReports>
+    <enableSearch>true</enableSearch>
+    <enableSharing>true</enableSharing>
+    <enableStreamingApi>true</enableStreamingApi>
+    <fields>
+        <fullName>Contact__c</fullName>
+        <deleteConstraint>SetNull</deleteConstraint>
+        <description>The person that is speaking this language.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The person that is speaking this language.</inlineHelpText>
+        <label>Contact</label>
+        <referenceTo>Contact</referenceTo>
+        <relationshipLabel>Contact Languages</relationshipLabel>
+        <relationshipName>Contact_Languages</relationshipName>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Lookup</type>
+    </fields>
+    <fields>
+        <fullName>Language__c</fullName>
+        <deleteConstraint>SetNull</deleteConstraint>
+        <description>The language this contact is associated with.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The language this contact is associated with.</inlineHelpText>
+        <label>Language</label>
+        <referenceTo>Language__c</referenceTo>
+        <relationshipLabel>Contact Languages</relationshipLabel>
+        <relationshipName>Contact_Languages</relationshipName>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Lookup</type>
+    </fields>
```

Comment:
@robertbwatson -- are we not also providing a picklist for fluency here? I review the document and it looks like we decided that we may have some EdFi specific values (for certification) but I still think we want the field on the object in HEDA with perhaps "Conversational|Can Read" or something similar.

Reply:
@lifewithryan Sorry, I created these pull requests as "draft" PRs and should have been more clear that I did so - I fully intend to make more modifications here and it is not ready for review. (Just announced recently - https://github.blog/2019-02-14-introducing-draft-pull-requests/)

Reply:
AHHHHH :) thanks